1_wqpdata.yml

```diff
@@ -67,42 +68,46 @@ targets:
   # remake targets, but let's risk having to redo the inventory for the sake of
   # keeping this remake file a touch simpler and practicing the two-target option
   # for gd_put/gd_get)
-  1_wqpdata/out/inventory/wqp_inventory.feather.ind:
-    depends: 1_wqpdata/out/inventory
-    command: inventory_wqp(
-      ind_file=target_name,
-      wqp_state_codes=wqp_state_codes,
-      wqp_states=wqp_states,
-      wqp_codes=wqp_codes)
-  # the only job of this data recipe is to pull data from the shared cache
-  1_wqpdata/out/inventory/wqp_inventory.feather:
-    command: gd_get('1_wqpdata/out/inventory/wqp_inventory.feather.ind')
-  # use the inventory. because this is an object, everybody will end up
-  # pulling wqp_inventory.feather and building this object locally, if only
-  # to know whether 1_wqpdata/log/tasks_1_wqp.ind is up to date
-  wqp_pull_partitions:
-    command: partition_inventory(
-      inventory_ind='1_wqpdata/out/inventory/wqp_inventory.feather.ind',
-      wqp_pull=wqp_pull,
-      wqp_state_codes=wqp_state_codes,
-      wqp_codes=wqp_codes)
-
-  # -- pull the data --
+
+  # prepare a remake-style plan for inventorying each constituent as a separate
+  # remake target in a separate remake file (tasks_1_wqp_inventory.yml)
+  inventory_plan:
+    command: plan_inventory(constituents=wqp_codes, folders=wqp_pull_folders)
+  tasks_1_wqp_inventory.yml:
+    command: create_task_makefile(
+      makefile=target_name,
+      task_plan=inventory_plan,
+      include='1_wqpdata.yml',
+      packages=I(c('dplyr', 'dataRetrieval', 'feather', 'scipiper')),
```

Comment:
Does remake let you do this - include a `c()` within an `I()` within a function call? If so, cool! I thought remake was more limiting than that, which is how I often end up writing my own wrapper for create_task_makefile for each specific makefile I need.